Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM alpine:latest
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
 
 WORKDIR /
 
-RUN apk add -U --no-cache ca-certificates xfsprogs xfsprogs-extra
-
 COPY direct-csi /direct-csi
 COPY CREDITS /licenses/CREDITS
 COPY LICENSE /licenses/LICENSE
 
-ENTRYPOINT ["/direct-csi"]
+COPY centos7.repo /etc/yum.repos.d/centos7.repo
+
+RUN \
+    curl https://www.centos.org/keys/RPM-GPG-KEY-CentOS-7 -o /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+    mv /etc/yum.repos.d/ubi.repo /etc/yum.repos.d/ubi.repo.old  && \
```

Comment:
Renaming ubi.repo is not correct enable relevant repos such that we can also get security fixes for other packages, disabling this means that future updates to packages are disabled.